File: php/2nd_week/Inheritance/Manusia.php
<?php
class Manusia{
    private $nama;
    private $alamat;
    private $no_ktp, $no_telepon;

    function __construct()
    {
    }

    function setNama($nama){
        $this->nama = $nama;
    }

    function setAlamat($alamat){
        $this->alamat = $alamat;
    }

    function setNoKtp($no_ktp){
        $this->no_ktp = $no_ktp;
    }

    function setNoTelepon($no_telepon){
        $this->no_telepon = $no_telepon;
    }

    function getNama(){
        return $this->nama;
    }

    function getAlamat(){
        return $this->alamat;
    }

    function getNoKtp(){
        return $this->no_ktp;
    }

    function getNoTelepon(){
        return $this->no_telepon;
    }

    function __destruct()
    {
    }
}
?>

File: cpp/2nd_Week/Inheritance/Manusia.cpp
#include <iostream>
#include <string>
using namespace std;

class Manusia
{
    private:
        string name;
        string alamat;
        string no_ktp;
        string no_telepon;

    public:
        Manusia(){
        };

        //setter method
        void setNama(string name){
            this->name = name;
        }

        void setAlamat(string alamat){
            this->alamat = alamat;
        }

        void setNoKtp(string no_ktp){
            this->no_ktp = no_ktp;
        }

        void setNoTelepon(string no_telepon){
            this->no_telepon = no_telepon;
        }

        //getter method
        string getNama(){
            return this->name;
        }

        string getAlamat(){
            return this->alamat;
        }

        string getNoKtp(){
            return this->no_ktp;
        }
        
        string getNoTelepon(){
            return this->no_telepon;
        }

        ~Manusia(){
        };
};



